server/block/break_info.go

```diff
@@ -85,6 +85,8 @@ type BreakInfo struct {
 	Drops func(t item.Tool, enchantments []item.Enchantment) []item.Stack
 	// XPDrops is the range of XP a block can drop when broken.
 	XPDrops XPDropRange
+	// BlastResistance ...
+	BlastResistance float64
```

Comment:
This should be documented. Also maybe we should default this to a number that is based on the hardness? I think that applies to the majority of the blocks.